packages/manager/modules/sms/package.json

```diff
@@ -30,7 +30,9 @@
     "moment": "^2.22.2"
   },
   "devDependencies": {
-    "@ovh-ux/component-rollup-config": "5.0.0-beta.5"
+    "@ovh-ux/component-rollup-config": "5.0.0-beta.5",
+    "bootstrap": "~3.3.7",
```

Comment:
Theses `devDependencies` aren't linked to the new feature. It should be added in a separate commit.